v1/atm/atm.gunk

```diff
@@ -203,6 +211,11 @@ type ATMService interface {
 	//         Responses: map[string]openapiv2.Response{
 	//                 "201": openapiv2.Response{
 	//                         Description: "Transaction created successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/atmCreateATMResponse",
```

Comment:
Should the Ref be "#/definitions/atm`s`CreateATMResponse"?

Reply:
https://github.com/openbank/openbank/pull/17#discussion_r314579305